govc/test/host.bats

```diff
@@ -80,6 +79,28 @@ load test_helper
   assert_success
 }
 
+@test "host maintanence vc" {
+  vcsim_env
+
+  run govc host.info
+  assert_success
+  grep -q -v Maintenance <<<$output
```

Comment:
Can you quote these `"$output"`

I know some existing tests don't quote $output, but that was before we discovered shellcheck, see:
```/usr/local/libexec/bats-preprocess < host.bats | shellcheck -```

Been trying to keep new tests shellcheck clean where possible.